packages/reakit/src/Menu/MenuItem.ts

```diff
@@ -0,0 +1,20 @@
+import { ifProp, theme } from "styled-tools";
+import styled from "../styled";
+import as from "../as";
+import Box, { BoxProps } from "../Box";
+
+type Props = {
```

Comment:
```suggestion
export interface MenuItemProps extends BoxProps {
```
See https://github.com/reakit/reakit/blob/master/CONTRIBUTING.md#components